### PR TITLE
Issue #SC-1087 Fix readme

### PR DESCRIPTION
**Pull request by indrajra**
Appears we have the user/org schemas in two places - 1) sb_registry folder (source) and 2) in java/main/resources/schemas. Both of them are checked in. So, checking in 2) files as well to maintain. Perhaps we can shed this away and retain only the sb_registry.

Using the application.yml.sample, one creates the application.yml (not checked-in), so that it allows developers to change their config locally (passwords, URLs etc).

Fixed README to clearer messaging.

### CODE REVIEW

File: java/registry/src/main/resources/public/_schemas/User.json [hunk 1]
```diff
@@ -138,8 +138,8 @@
 
     "privateFields": ["phone", "email", "userName", "status", "loginId"],
     "signedFields": [],
-    "indexFields": ["email","status","userName","phone","loginId"],
-    "uniqueIndexFields": ["email","phone","loginId"],
+    "indexFields": ["email","status","userName","phone"],
+    "uniqueIndexFields": ["email","phone"],
```
Comment (manzarul) [inline java/registry/src/main/resources/public/_schemas/User.json]: userName must be unique.

File: java/registry/src/main/resources/public/_schemas/User.json [hunk 2]
```diff
@@ -138,8 +138,8 @@
 
     "privateFields": ["phone", "email", "userName", "status", "loginId"],
```
Comment (manzarul) [inline java/registry/src/main/resources/public/_schemas/User.json]: loginId is not used by system now. we can remove it.

File: java/registry/src/main/resources/public/_schemas/UserFieldStatus.json [hunk 3]
```diff
@@ -66,6 +66,8 @@
 
     "privateFields": [],
     "signedFields": [],
+    "indexFields": ["loginId"],
+    "uniqueIndexFields": ["loginId"],
     "systemFields": ["createdOn", "updatedAt", "createdBy", "updatedBy"]
```
Comment (manzarul) [inline java/registry/src/main/resources/public/_schemas/UserFieldStatus.json]: use same semantic :  some place we have createdAt and here we have createdOn.

File: java/registry/src/main/resources/application.yml.sample [hunk 4]
```diff
@@ -157,7 +148,7 @@ signature:
     #file: ${audit_frame_file:audit_frame.json}
 
 authentication:
-  enabled: ${authentication_enabled:true}
+  enabled: ${authentication_enabled:false}
```
Comment (manzarul) [inline java/registry/src/main/resources/application.yml.sample:151]: is this value need to be put into system ENV or properties file.?

**Review by manzarul — APPROVED** [review]

File: java/registry/src/main/resources/public/_schemas/UserFieldStatus.json (hunk 3 above)
Comment (indrajra) [reply java/registry/src/main/resources/public/_schemas/UserFieldStatus.json]: done.

File: java/registry/src/main/resources/application.yml.sample (hunk 4 above)
Comment (indrajra) [reply java/registry/src/main/resources/application.yml.sample:151]: Not a must, but can be overridden with env var.

**Pull request merged**